h2o-r/h2o-package/R/models.R

```diff
@@ -2584,9 +2593,13 @@ h2o.get_regression_influence_diagnostics <- function(model, predictorSize = -1)
 #' }
 #' @export 
 h2o.negative_log_likelihood <- function(model) {
-    warning_message <- paste("This is the simplified negative log likelihood function used during training for speedup.",
-                             "To see the correct values (for loglikelihood and AIC), set calc_like=True and call", 
-                             "model.model_performance().loglikelihood() and model.model_performance().aic()")
+    if (model@allparameters$calc_like) {
+        warning_message <- paste("This is the simplified negative log likelihood function used during training for speedup.",
+                                 "To see the correct value call h2o.loglikelihood(model)")
```

Comment:
Should we end the sentence here? I don't have a strong opinion but the dot is in the previous message after `h2o.aic().` (line 2014)
```suggestion
                                 "To see the correct value call h2o.loglikelihood(model).")
```